sys/riscv/conf/CHERI-PURECAP-QEMU-MFS-ROOT

```diff
@@ -0,0 +1,10 @@
+#NO_UNIVERSE
+
+include "QEMU"
+include "std.CHERI-PURECAP"
+include "std.MFS-ROOT"
+
+options 	ENABLE_PAST_LOCAL_VULNERABILITIES
+options 	ENABLE_PAST_REMOTE_VULNERABILITIES
```

Comment:
```suggestion
include "CHERI-PURECAP-QEMU"
include "std.MFS-ROOT"

```

Reply:
I guess we only care about those in FETT-specific configurations?

Reply:
CHERI-PURECAP-QEMU includes them as well as std.CHERI-PURECAP.